.github/workflows/integration.yml

```diff
@@ -34,3 +32,18 @@ jobs:
     - if: matrix.os == 'ubuntu-latest' && matrix.python-version == 3.8
       name: Upload codecov report
       uses: codecov/codecov-action@v2
+
+  integration-python3-11:
```

Comment:
I think we can probably just bump this library to 3.10 only. It might be complicated or strange to have more support on only some operating systems